includes/apple-exporter/class-theme.php

```diff
@@ -832,6 +833,17 @@ private static function initialize_options() {
 				'label'   => __( 'Dropcap font face', 'apple-news' ),
 				'type'    => 'font',
 			],
+			'dropcap_minimum'                    => [
+				'default' => 100,
+				'label'   => __( 'Minimum number of characters for dropcap to take affect.', 'apple-news' ),
+				'type'    => 'integer',
+			],
+			'dropcap_minimum_opt_out'            => [
+				'default' => 'no',
+				'label'   => __( 'Opt out of minimum character dropcap behavior.', 'apple-news' ),
```

Comment:
Because the conditions are more than just character limits, they're also punctuation, recommend updating this label to be more clear

```suggestion
				'label'   => __( 'Opt out of conditional dropcap behavior.', 'apple-news' ),
```